packages/react/src/components/ToggleSmall/ToggleSmall.Skeleton.js

```diff
@@ -10,7 +10,7 @@ import React from 'react';
 import cx from 'classnames';
 import { PrefixContext } from '../../internal/usePrefix';
 
-export default class ToggleSmallSkeleton extends React.Component {
+export class ToggleSmallSkeleton extends React.Component {
```

Comment:
Same question/comment here about having the export statements at the bottom of the file.